src/openzaak/js/components/admin/zaaktypen/selectielijst.js

```diff
@@ -60,11 +60,14 @@ ProcestypeOptions.propTypes = {
 
 
 const renderProcestypeOptions = (root, initialValue, year) => {
-    ReactDOM.render(
+    if (!root._reactRoot) {
```

Comment:
Not sure if it's related to this, but if I change the year of the selectielijst, for some reason the help icon is duplicated and also shows up inside the `Selectielijst procestype` label text:

<img width="658" height="149" alt="afbeelding" src="https://github.com/user-attachments/assets/d266c6d1-bbd0-43e9-9542-addd009a8b87" />

Can you check if this can be fixed?